docs/guides/integration/github.md

```diff
@@ -267,6 +267,30 @@ Its effect on performance is dependent on the packages being installed.
 
     If using `uv pip`, use `requirements.txt` instead of `uv.lock` in the cache key.
 
+!!! warning
+
+    [post-job-hook]: https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/running-scripts-before-or-after-a-job
+
+    When using non-ephemeral self-hosted runners the default cache location can grow over time,
+    becoming problematic. In such cases its possible to set the caching location relative to the
+    GitHub Workspace and clean it once the job finishes using a [Post Job Hook][post-job-hook].
+
+    ```yaml
+    install_job:
+      env:
+        # Configure a relative location for the uv cache
+        UV_CACHE_DIR: ${{ github.workspace }}/.cache/uv
+      # ... restore uv cache ...
```

Comment:
What is being referred to here?

Reply:
Good question, it was a callout to the previous section https://docs.astral.sh/uv/guides/integration/github/#caching

Reply:
But would you actually restore the cache in this situation? My understanding is the cache is deleted at the end of each job so what is there to persist and restore? (This confusion has been why this pull request has been lingering, sorry!)

Reply:
Ah I see. That was meant to be optional step (if desired). We can easily remove it if its confusing.

Reply:
```suggestion
```